charts/pega/templates/pega-registry-secret.yaml

```diff
@@ -1,3 +1,19 @@
+{{- if .Values.global.docker.config.path }}
+kind: ExternalSecret
+apiVersion: kubernetes-client.io/v1
+metadata:
+  name: {{ template "pegaRegistrySecret" }}
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "5"
+spec:
+  type: kubernetes.io/dockerconfigjson
+  backendType: secretsManager
+  data:
+    - key: {{ .Values.global.docker.config.path }}
```

Comment:
The cassandra chart uses `pullSecrets` as the name for a similar thing. We could adopt a similar naming pattern here as well and do something like `docker.config.pullSecrets`. Just throwing it out for discussion related purposes as I'm not married to the name here.